OOP.Task4/Program.cs

```diff
@@ -5,13 +5,13 @@ namespace OOP.Task4
 {
     class Program
     {
-        static string Input { get; set; }
+      public static string Input { get; set; }
 
            Program()
            {
                Input = Program.InputAndValidation();
```

Comment:
since you're in the Program class, you don't need to call Program.InputAndValidation().
InputAndValidation() is enough